packages/ibm-products/src/components/PageHeader/PageHeaderTitle.js

```diff
@@ -67,6 +85,17 @@ export const PageHeaderTitle = ({ blockClass, hasBreadcrumbRow, title }) => {
             inheritTypography
             {...rest}
           />
+        ) : isEllipsisApplied ? (
+          <DefinitionTooltip
+            openOnHover={false}
+            align={'bottom'}
```

Comment:
Hi @elycheea , 
Thanks for looking into this. I was thinking like,  we are applying tooltip  only for long titles and has overflowed as well. That means there wouldn't be enough space at right side for tooltip. Also, probably,  there should be space constraints in top and left since its a page title,  leaving the only option left for us is `bottom` . Also please not that carbon is not supporting floating ui for Definition tooltips, so the tooltips may hide under parent divs. 
Can you share your thoughts?